core/pom.xml

```diff
@@ -36,26 +36,33 @@
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
It's not necessary the explicit reference anymore. It was included because of an old conflict.